mne/baseline.py

```diff
@@ -24,7 +24,7 @@ def _log_rescale(baseline, mode='mean'):
 
 
 @verbose
-def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
+def rescale(data, times, baseline, mode='mean', copy=None, verbose=None):
```

Comment:
Just need to add default=True to the _check_copy_dep, feel free to do it or I will in a few hours. 

I asked about the public thing earlier IIRC @agramfort or @dengemann use it in their code